### ISSUE TITLE
Error values from TS0221_plug

### ISSUE THREAD

**Issue by SargonofAssyria** [stas-demydiuk/domoticz-zigbee2mqtt-plugin#614]
I have this EU plug https://www.zigbee2mqtt.io/devices/TS0121_plug.html#tuya-ts0121_plug
This plug gives 2 messages right after each other. From Z2M is only 1 message, the first.
The first message changes when power changes, the second message is always 20 Watts.

2021-04-01 14:12:21.278  Status: dzVents: Info: Handling events for: "3D-Printer kWh Meter", value: "12;10752.4"
2021-04-01 14:12:21.485  Status: dzVents: Info: Handling events for: "3D-Printer kWh Meter", value: "20;10752.4"

**Additional information**
Zigbee2MQTT version 1.18.1-dev commit: ddbfa02
Coordinator type zStack3x0
Coordinator revision 20210120
Frontend version 0.3.113
Plugin version: 3.0.0 (the latest)

P.S. Found out that a have a second device with almost the same issue, this time is the second wattage 30 Watts.

**Comment by SargonofAssyria** [comment]
Found the cause: It is all about the "power_outage_memory"
If that is set to "On" the second message is 10 Watts, if set to "Off" 20 Watts and set to "Restore" it is 30 Watts.
I think it is about "power" and "power_outage_memory", so I think it can be easily solved.
I don't know how, but hope that somebody can do that.

**Comment by SargonofAssyria** [comment]
An other thing I noticed is that the "power_outage_memory" is defined as an On/Off switch
This plug should be a selector switch as seen here. I sthat the reason this this device is not created?
![2021-04-03_09-50-54](https://user-images.githubusercontent.com/24824113/113472238-272e8c80-9462-11eb-8633-d94da58c1803.png)

**Comment by SargonofAssyria** [comment]
After a lot fiddling around, I suddenly do not got the "power_outage_memory" message from 1 device anymore, and instantly my power consumption device in Domoticz work as it should. 2 Other device still have that message and don't work as expected.

**Comment by SargonofAssyria** [comment]
After a lot of removing and including the last problematic device I got that working too.
I noticed that "power_outage_memory" is not included in the mqtt messages anymore.
Previous there was NO "energy" in the mqtt message, and now it is.
I cannot find out what changed the device for putting out "energy" and NOT "power_outage_memory", but that's the configuration that works, hopefully forever.
